test/k8sT/Updates.go

```diff
@@ -235,17 +234,19 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldHelmChartVers
 		cleanupCiliumState(filepath.Join(kubectl.BasePath(), helpers.HelmTemplate), newHelmChartVersion, "", newImageVersion, "")
 
 		By("Cleaning Cilium state (%s)", oldImageVersion)
-		cleanupCiliumState("cilium/cilium", oldHelmChartVersion, "quay.io/cilium/cilium", oldImageVersion, "")
+		cleanupCiliumState("cilium/cilium", oldHelmChartVersion, "quay.io/cilium/cilium-ci", oldImageVersion, "")
 
 		By("Deploying Cilium %s", oldHelmChartVersion)
 
 		opts := map[string]string{
-			"image.tag":                     oldImageVersion,
-			"operator.image.tag":            oldImageVersion,
-			"hubble.relay.image.tag":        oldImageVersion,
-			"image.repository":              "quay.io/cilium/cilium",
-			"operator.image.repository":     "quay.io/cilium/operator",
-			"hubble.relay.image.repository": "quay.io/cilium/hubble-relay",
+			"image.tag":                              oldImageVersion,
+			"operator.image.tag":                     oldImageVersion,
+			"hubble.relay.image.tag":                 oldImageVersion,
+			"clustermesh.apiserver.image.tag":        oldImageVersion,
+			"image.repository":                       "quay.io/cilium/cilium-ci",
+			"operator.image.repository":              "quay.io/cilium/operator",
```

Comment:
I'm surprised the test is passing. Shouldn't we need to pass `operator.image.suffix=-ci`? :thinking:

Reply:
` CILIUM_OPERATOR_SUFFIX=-ci` is passed in from jenkins (seen in `test-output.log` from the focused run).

I believe that this will always be set when the upgrade tests runs:

Matches for operator suffix option:
```
$ git grep -l operator-suffix jenkinsfiles/
jenkinsfiles/ginkgo-gke.Jenkinsfile
jenkinsfiles/ginkgo-kernel.Jenkinsfile
```

Files lacking the option:
```
$ git grep -L operator-suffix jenkinsfiles/
jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile
jenkinsfiles/kubernetes-upstream.Jenkinsfile
```